### DISCUSSION TITLE
zh/snippets/

### DISCUSSION THREAD

**Discussion by giscus** [iluwatar/java-design-patterns-vuepress-web#57] [Announcements]
# zh/snippets/

30秒快速编码 灵感来源: 30秒快速编码, 这是一组可重复使用的、经过测试的、可粘贴的Java17兼容代码片段，您可以在30秒内或更短的时间内理解这些代码片段。 如果您有兴趣对此库进行贡献，请参阅 贡献说明.

https://java-design-patterns.com/zh/snippets/

**Comment by hjg66-5** [comment]
6啊

**Reply by MoFany** [reply]
OK，可以挺好的

**Comment by wuxin0011** [comment]
好

**Comment by suhaoxu** [comment]
666
